src/python/WMCore/ReqMgr/Service/Request.py

```diff
@@ -319,7 +322,12 @@ def get(self, **kwargs):
         result = self._intersection_of_request_info(request_info)
         if len(result) == 0:
             return []
-        return rows([result])
+        
+        if common_dict == 1:
+            response_list = result.values()
```

Comment:
Yes, we need to know what exact information People need by inputdatasets, outputdatasets, etc. We can put some minimal information first as Alan suggested. If users need more we can add accordingly.